test-harness.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+START=$(date "+%s")
+
+set -e
+
+ENV_FILE=".test-env"
+source $ENV_FILE
+
+rootdir=$(dirname "$0")
+pushd "$rootdir"
+
+## Reset test harness
+if [ -d "$SDK_TESTING_HARNESS" ]; then
+  pushd "$SDK_TESTING_HARNESS"
+  ./scripts/down.sh
+  popd
+  rm -rf "$SDK_TESTING_HARNESS"
+else
+  echo "test-harness.sh: directory $SDK_TESTING_HARNESS does not exist - NOOP"
+fi
+
+git clone --single-branch --branch "$SDK_TESTING_BRANCH" "$SDK_TESTING_URL" "$SDK_TESTING_HARNESS"
+
+## OVERWRITE incoming .env with .test-env
+cp "$ENV_FILE" "$SDK_TESTING_HARNESS"/.env
+
+## Copy feature files into the project resources
+mkdir -p tests/features
```

Comment:
Should there be a line to remove any stale tests? If we are going for a "fresh" `make` and `test`, it's nicer to remove existing tests:

```
rm -rf tests/features
```

Reply:
Good catch!